wasm/js/webrtc.js

```diff
@@ -109,6 +181,26 @@
 			}
 		},
 
+		rtcLocalDescriptionSdp: function(pc) {
```

Comment:
Could you please rename `rtcLocalDescriptionSdp` to `rtcGetLocalDescription` to match the naming in the C API of libdatachannel?